docs/Deployment.md

```diff
@@ -240,15 +233,15 @@ The address and port on which the node will be advertised. Other nodes must be a
 
 The name of the Raft log file.
 
-RedisRaft uses this as the base name of the Raft log files, and creates additional files including `<filename>.idx`, `<filename>.tmp.idx` and `<filename>.tmp`.
+RedisRaft uses this as the base name of the Raft log files, and creates additional files including `<filename>.idx`, `<filename>.tmp.idx`, `<filename>.tmp`, `<filename>.meta` and `<filename>.meta.tmp`.
 
 *Default*: `redisraft.db.`
 
 ### `periodic-interval`
 
 The number of milliseconds between internal RedisRaft cluster events such as heartbeats, message retransmissions, and re-election announcements.
 
-If the `request-timeout` or `election-timeout` values must be reduced for faster failure detection, you'll also want to reduce this value as well. However, reducing this value will result with more network traffic.
```

Comment:
raft library does not send messages blindly anymore (it is sent only if it is required).